lab_2_tokenize_by_bpe/main.py

```diff
@@ -47,6 +77,31 @@ def merge_tokens(
     :param pair: a pair of tokens to be merged
     :return: dictionary in the form of <preprocessed word: number of occurrences>
     """
+    if (not isinstance(word_frequencies, dict)
+       or not isinstance(pair, tuple)):
+        return None
+    new_dict = {}
+    for word in word_frequencies:
+        new_word = []
+        ''' Alternative variation with a switch:
```

Comment:
the actual and alternative implementations are strange. Wee need the third

Reply:
the third has been introduced